use_cases/timeseries.rst

```diff
@@ -276,43 +275,16 @@ Archiving a Row Partition to Columnar Storage
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 When a row partition has filled its range, you can archive it to compressed
-columnar storage. The process is:
-
-1. Make a columnar copy of the row partition.
-2. Detach the row partition.
-3. Perform table renames.
-4. Attach the columnar copy in the row partition's stead.
-
-In code, here's how to turn ge3 columnar:
+columnar storage.
 
 .. code-block:: postgresql
 
-  BEGIN;
-  
-  -- uncomment the following statement to avoid deadlock risk
-  -- at the cost of holding the lock during the data conversion:
-  --   LOCK TABLE github.columnar_events IN ACCESS EXCLUSIVE MODE;
-  
-  LOCK TABLE ge3 IN EXCLUSIVE MODE;
-  CREATE TABLE ge3_tmp_new(LIKE ge3) USING columnar;
-  INSERT INTO ge3_tmp_new SELECT * FROM ge3;
+  SELECT alter_table_set_access_method('ge0', 'columnar');
```

Comment:
That's handy. :)

Any reason the example now uses ge0, while it was using ge3 before? Presumably the older partitions like ge3 would already be columnar?

Reply:
Changed to ge3